Doc/tools/static/changelog_search.js

```diff
@@ -1,53 +1,56 @@
-$(document).ready(function() {
+document.addEventListener('DOMContentLoaded', function() {
     // add the search form and bind the events
-    $('h1').after([
+    document.querySelector('h1').insertAdjacentHTML('afterend', [
       '<p>Filter entries by content:',
       '<input type="text" value="" id="searchbox" style="width: 50%">',
       '<input type="submit" id="searchbox-submit" value="Filter"></p>'
     ].join('\n'));
 
     function dofilter() {
+        let query;
         try {
-            var query = new RegExp($('#searchbox').val(), 'i');
+            query = new RegExp(document.querySelector('#searchbox').value, 'i');
         }
         catch (e) {
             return; // not a valid regex (yet)
         }
         // find headers for the versions (What's new in Python X.Y.Z?)
-        $('#changelog h2').each(function(index1, h2) {
-            var h2_parent = $(h2).parent();
-            var sections_found = 0;
+        const h2s = document.querySelectorAll('#changelog h2');
+        for(let h2 of h2s) {
```

Comment:
`const` is usually preferred if the variable doesn't change
```suggestion
        for(const h2 of h2s) {
```